pom.xml

```diff
@@ -266,23 +201,21 @@
       <version>1.16</version>
     </dependency>
 
-
     <dependency>
       <groupId>junit</groupId>
       <artifactId>junit</artifactId>
-      <version>4.12</version>
       <scope>test</scope>
     </dependency>
     <dependency>
       <groupId>org.mockito</groupId>
-      <artifactId>mockito-all</artifactId>
+      <artifactId>mockito-core</artifactId>
       <version>1.10.19</version>
       <scope>test</scope>
     </dependency>
     <dependency>
       <groupId>org.apache.httpcomponents</groupId>
       <artifactId>httpclient</artifactId>
-      <version>4.4.1</version>
+      <version>4.5</version>
```

Comment:
I thought there were significant and relevant changes between httpclient 4.4 and 4.5.  Have you performed interactive testing of the git server plugin with this change?  

The git server plugin provides a git repository interface to the userContent directory in JENKINS_HOME.